### ISSUE TITLE
Media Module - Unable to remove linked media file

### ISSUE THREAD

**Issue by stanbluijs** [AsgardCms/Platform#123]
First of all, keep up the good work!
I'm working on my first AsgardCMS project.

Development system:
OS - Windows 10
Using - XAMMP v3.2.1
PHP version - 5.6.3
Browser: FireFox, Chrome both latest version

First I had the same issue that the trait was not working (https://github.com/AsgardCms/Platform/issues/91) but I learned that you have to actualy 'use' the trait by adding 'use MediaRelation;' inside your entity.
Maybe it is an idea to add this in the documentation for the people (like me) who are not used to use a trait.

The issue I'm having is:

After successfully linking a media file to an record it will be displayed as it should. But when trying to delete it is not working (because you selected the wrong file), the red cross button (pointed in screen-1) will add a hashtag to the URL, but the file is not unlinked in database and stays visible on screen.
I was not able to find out why this happens, no JS erros on the page and also no errors after clicking the remove button.
![screen-1](https://cloud.githubusercontent.com/assets/7602123/10980550/d31a83f8-8402-11e5-8046-e457ae479e71.jpg)

After the media file is linked to the record, and editing the record for the second time, the linked image is not being displayed. I'm only not sure if this supposed to happen. But if not, how is it then possible to unlink the media file from the record?
![screen-2](https://cloud.githubusercontent.com/assets/7602123/10980540/c4e63fde-8402-11e5-9389-94c6af26812d.jpg)

**Comment by nWidart** [comment]
Hi,

Do you get an error in console when pressing the `x` ?

**Comment by stanbluijs** [comment]
Hi,
Thank you for your reply.
I do not get any error in console while pressing the X. It is only adding a hashtag after the current URL.

**Comment by nWidart** [comment]
Hello, 

I just created a clean module to try to reproduce this error.

So I added the partial on the view, added an image. The image was correctly stored in the `media__imageables` table. 
Next I reloaded the page, image still there.
Next I pressent the `x` icon, and like you reported nothing happened. 

I want in the partial itself, everything looks good. So I added a `console.log('ok')` in the `$('.jsRemoveLink').on('click'` callback, and tried again. This was to check if this was correctly triggered.

***

I now think I understand the issue, the issue occurs when added an image, and removing it directly without refreshing the page before. 
This is caused because I don't bind the clic event on the window, and the images are added on the dom later.

Will check for a fix now. :)
